staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -1444,6 +1444,10 @@ func TestSlowNodeSync(t *testing.T) {
 		t.Fatalf("error creating node3, err: %v", err)
 	}
 
+	// Allow a bit of time for the informer cache to get populated with the new
+	// node
+	time.Sleep(20 * time.Millisecond)
```

Comment:
if we know is because we need to wait for node3 why don't we just poll the cache until we found the node there?

Reply:
Yeah, I guess that's better. I've updated the PR